src/infra/workload/releaseunit/modules/stamp/policy.tf

```diff
@@ -44,3 +44,12 @@ resource "azurerm_resource_group_policy_assignment" "no_privileged_containers" {
   policy_definition_id = "/providers/Microsoft.Authorization/policyDefinitions/95edb821-ddaf-4404-9732-666045e056b4"
   display_name         = "Kubernetes cluster should not allow privileged containers"
 }
+
+# Role-Based Access Control (RBAC) should be used on Kubernetes Services (BuiltIn)
+# https://github.com/Azure/azure-policy/blob/master/built-in-policies/policyDefinitions/Security%20Center/ASC_EnableRBAC_KubernetesService_Audit.json
+resource "azurerm_resource_group_policy_assignment" "aks_rbac_enabled" {
```

Comment:
why do we need this policy? cant we just enable RBAC on our AKS cluster?

Reply:
its just an audit policy - the set of policies we've in here are just reflecting some best practices and policies we've seen in the field - and rbac is of course enabled (by default and explicit)

Reply:
hm but by that logic we would need to add a lot more policies?! and a policy which will just validate what we deploy ourselves (yes, AKS is RBAC on), doesn't add real value?!

Reply:
that's correct - its pretty much just a sample - and it allows us to show and demo the azure policy panel in the portal 

wasn't planning to add all policies

Reply:
since we already have a couple of example how to add policies, I would have dropped it. But I also understand if you want this one in. So your call :)

Reply:
Debateable which policies we want to keep / add / remove. Have no real preference here. Would like to keep some different ones for a proper demo, but thats it. 

CC: @hjscherer